easybuild/easyconfigs/n/NLTK/3.5-GCC-8.3.0-Python-3.7.4.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'PythonPackage'
+
+name = 'NLTK'
+version = '3.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.nltk.org/'
+description = "NLTK is a leading platform for building Python programs to work with human language data."
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_ZIP]
+checksums = ['845365449cd8c5f9731f7cb9f8bd6fd0767553b9d53af9eb1b3abf7700936b35']
+
+dependencies = [('Python', '3.7.4')]
+
```

Comment:
@levlafayette please add the following lines (you might have to add extensions if the build fails with this):
```
download_dep_fail = True
use_pip = True
sanity_pip_check = True
```